functions/function/http-request-function/src/main/java/org/springframework/cloud/fn/http/request/HttpRequestFunctionProperties.java

```diff
@@ -52,13 +52,6 @@ public class HttpRequestFunctionProperties {
 	 */
 	private long timeout = 30_000;
 
-	/**
-	 * Maximum buffer size in bytes allocated for input stream buffers. Defaults to 256k.
-	 * Increase, as necessary, for posting or getting large binary content.
-	 */
-
-	private int maximumBufferSize = 256 * 1024;
```

Comment:
I typically would recommend that we add a unit test to cover this removal but there is currently no coverage for this particular property as it is difficult to verify anyways. Also, we can trust that the WebClient spring.codecs property has been thoroughly tested.